### DISCUSSION TITLE
[Script request] Speedtest tracker

### DISCUSSION THREAD

**Discussion by Aloe-recite** [community-scripts/ProxmoxVE#118] [Request script]
### Application Name

Speedtest Tracker

### Website

https://github.com/alexjustesen/speedtest-tracker

### Description

Speedtest Tracker is a self-hosted internet performance tracking application that runs speedtests using Ookla's Speedtest service.
The main use case for Speedtest Tracker is to build a history of your internet's performance so that you can be informed when you're not receiving your ISP's advertised rates.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Why not MySpeed LXC? Runs @my Homelab since 5 or 6 month and worked perfectly

**Reply by Aloe-recite** [reply]
> Why not MySpeed LXC? Runs @my Homelab since 5 or 6 month and worked perfectly

Yes it works perfectly and still have it on my system, however Speedtest Tracker has more features (the most useful ones are notifications with a plethora of operatros when under certain thresholds, which is great for my not-so-stable ISP), although also some drawbacks and limitations, and, IMHO, a nicer UI overall (personal preference though). I currently have it on my Docker test systems and seems to work great, although there's room or improvement. My 2c of course!

**Comment by drVLecter** [comment]
+1 for this one.

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1169
